test/ctia/features_service_test.clj

```diff
@@ -0,0 +1,40 @@
+(ns ctia.features-service-test
+  (:require  [clojure.test :refer [deftest are is join-fixtures testing use-fixtures]]
+             [ctia.test-helpers.core :as th]))
+
+(defn- properties-fixture [t]
+  (th/with-properties
+    ["ctia.features.disable" "asset,actor,sighting"]
```

Comment:
Because the fixture doesn't merely setup these tests but also affects their outcome, I think that these properties are better inlined in each deftest.

That way it becomes evident that there's a cause-effect relationship between the chosen properties, and the enabled/disabled properties.

And importantly, one can write a more robust test. Sample:

```clj
(testing "When the properties are set to `x,y,z`, x+y+z are disabled"
 ...)
(testing "When the properties are set to `a,b,c`, a+b+c are disabled"
 ...)
```

with a test that shows that changing the properties changes the outcome, we can be sure that the test is passing for the right reason.